odoo_module_migrate/__main__.py

```diff
@@ -101,6 +101,19 @@ def get_parser():
         " the changes. (using git add and git commit command)"
     )
 
+    # TODO: Move to `argparse.BooleanOptionalAction` once in Python 3.9+
+    feature_parser = main_parser.add_mutually_exclusive_group(required=False)
+    feature_parser.add_argument(
+        "-pc", "--pre-commit", dest="pre_commit", action="store_true",
+        help="Run pre-commit and auto-commit the changes before performing"
+        " the migration (Enabled by default)",
+    )
```

Comment:
Why do you need this option if it is enabled by default?

Reply:
Well, true, I thought it would make it clearer for the CLI interface, but it just complicates... Already changed